terraform/gke/README.md

```diff
@@ -219,8 +219,7 @@ In this section you prepare your project for deployment.
     export TF_VAR_firestore_state_database=<DATABASE_NAME>
     ```
 
-    If you do not set this variable, the default database will be used
-    (`(default)`).
+    If you do not set this variable, Terraform will create and use a dedicated Firestore database named `memorystore-autoscaler-state` for storing Autoscaler state.
```

Comment:
Looks good but the `mdlint` check is [flagging the line length](https://github.com/GoogleCloudPlatform/memorystore-cluster-autoscaler/actions/runs/13576567730/job/37984028713#step:9:12), would you mind wrapping this to 80 chars? Thanks!

Reply:
(You can run this locally with `npm run mdlint`, if this is helpful).